### ISSUE TITLE
Swagger implementation

### ISSUE THREAD

**Issue by Dindaleon** [lukeed/polka#130]
Is it possible to have a swagger plugin for polka? Or maybe an alternative...

**Comment by lukeed** [comment]
Yes, there will be an OpenAPI integration, but not as part of 1.0.

I can't tell if it makes sense as a plugin or if it should be a larger framework that builds upon Polka. I'm leaning towards the latter because in order for OpenAPI to make sense (and work) it has to be strictly implemented.

**Comment by jgod** [comment]
A first step could be getting a "plugin" for https://github.com/PayU/openapi-validator-middleware, which already supports express, koa, and fastify. It looks like we're missing `req.baseUrl` and `req.route.path` that it relies on from express.

**Comment by jgod** [comment]
(I have it that library working with this setup)
